lighthouse-core/audits/byte-efficiency/unused-javascript.js

```diff
@@ -111,21 +208,29 @@ class UnusedJavaScript extends ByteEfficiencyAudit {
     }
 
     const items = [];
-    for (const [url, scripts] of scriptsByUrl.entries()) {
+    for (const [url, scriptCoverage] of scriptsByUrl.entries()) {
       const networkRecord = networkRecords.find(record => record.url === url);
       if (!networkRecord) continue;
-      const wasteData = scripts.map(UnusedJavaScript.computeWaste);
-      const item = UnusedJavaScript.mergeWaste(wasteData, networkRecord);
+      const wasteData = scriptCoverage.map(UnusedJavaScript.computeWaste);
+      const lengths = UnusedJavaScript.determineLengths(wasteData, networkRecord);
+      const bundle = bundles.find(b => b.script.src === url);
+      const item = UnusedJavaScript.mergeWaste(wasteData, networkRecord.url, lengths);
       if (item.wastedBytes <= IGNORE_THRESHOLD_IN_BYTES) continue;
+      if (bundle) {
+        UnusedJavaScript.createBundleMultiData(
+          item, wasteData, bundle, lengths, bundleSourceUnusedThreshold);
+      }
       items.push(item);
     }
 
     return {
       items,
       headings: [
-        {key: 'url', valueType: 'url', label: str_(i18n.UIStrings.columnURL)},
-        {key: 'totalBytes', valueType: 'bytes', label: str_(i18n.UIStrings.columnSize)},
-        {key: 'wastedBytes', valueType: 'bytes', label: str_(i18n.UIStrings.columnWastedBytes)},
+        /* eslint-disable max-len */
+        {key: 'url', valueType: 'url', subRows: {key: 'sources', valueType: 'code'}, label: str_(i18n.UIStrings.columnURL)},
```

Comment:
if the valueType can be `url` then it'd fix our text wrapping situation here: 
![image](https://user-images.githubusercontent.com/39191/75083733-ed096200-54cf-11ea-901e-240eb6022fc5.png)


repro: `node lighthouse-cli https://elsoberano.org/ --config-path lighthouse-core/config/experimental-config.js --only-audits unused-javascript -GA`

Reply:
I think we can safely use `url`
![image](https://user-images.githubusercontent.com/39191/75083861-d0b9f500-54d0-11ea-98bd-153b4a4d947e.png)

thanks to this code:
![image](https://user-images.githubusercontent.com/39191/75083850-b97b0780-54d0-11ea-810a-1d1e989c5058.png)


--------

i somewhat remember discussing how we handle sourcesRoot and doing a `new URL(url, base)` thing so that'd definitely play a role here for relative source paths.....

Reply:
i think amp is very unusual in how they do their maps.

i think the goal here is to reduce the length of each sub row...for that, we can trim off the longest common prefix (LCP ;) as we collect the source map. all other uses of maps will benefit from this too.

Reply:
sg!